packages/web/localizations/zh-cn.json

```diff
@@ -304,7 +304,10 @@
       "mobileMenu": "排序"
     },
     "MAX": "最大",
-    "searchTokens": "搜索代币"
+    "searchTokens": "搜索",
+    "selectToken": {
+      "title": "选择一个代币"
```

Comment:
Could you merge from master and add Taiwanese? I think a recent PR added Taiwanese to master branch.